apps/condo/migrations/20240405125421-0378_remove_bankcontractoraccounthistoryrecord_relatedtransactions_and_more.js

```diff
@@ -0,0 +1,266 @@
+
+exports.up = async (knex) => {
+    await knex.raw(`
+    BEGIN;
```

Comment:
Maybe add a statement timeout here? That's a lot of data to remove

Reply:
@reactpro said, that its ok, since drop column = mark at as hidden. So it's fast. And then garbage collector clears it in background